airbyte-cdk/python/airbyte_cdk/sources/streams/http/auth/token.py

```diff
@@ -36,3 +42,14 @@ def __init__(self, token: str, auth_method: str = "Bearer", auth_header: str = "
 
     def get_auth_header(self) -> Mapping[str, Any]:
         return {self.auth_header: f"{self.auth_method} {self._token}"}
+
+
+class MultipleTokenAuthenticator(HttpAuthenticator):
+    def __init__(self, tokens: str, auth_method: str = "Bearer", auth_header: str = "Authorization"):
```

Comment:
this should take a list of strings rather than a comma separated string

Reply:
good point, updated